src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java

```diff
@@ -266,9 +295,11 @@ private static String getRatioTapChangerFieldValue(FieldType field, @Nullable Ra
             return null;
         }
         return switch (field) {
-            case RATIO_TARGET_V -> String.valueOf(ratioTapChanger.getTargetV());
-            case LOAD_TAP_CHANGING_CAPABILITIES -> String.valueOf(ratioTapChanger.hasLoadTapChangingCapabilities());
-            case RATIO_REGULATION_MODE -> String.valueOf(getRatioRegulationMode(ratioTapChanger));
+            case RATIO_TARGET_V, RATIO_TARGET_V1, RATIO_TARGET_V2, RATIO_TARGET_V3 -> String.valueOf(ratioTapChanger.getTargetV());
+            case LOAD_TAP_CHANGING_CAPABILITIES, LOAD_TAP_CHANGING_CAPABILITIES_1, LOAD_TAP_CHANGING_CAPABILITIES_2,
+                 LOAD_TAP_CHANGING_CAPABILITIES_3 -> String.valueOf(ratioTapChanger.hasLoadTapChangingCapabilities());
+            case RATIO_REGULATION_MODE, RATIO_REGULATION_MODE_1, RATIO_REGULATION_MODE_2, RATIO_REGULATION_MODE_3
+                -> String.valueOf(getRatioRegulationMode(ratioTapChanger));
```

Comment:
In my opinion, it is better one line for each FieldType matched for visibility reason.

Reply:
do I repeat the line like 
```java
case RATIO_TARGET_V -> String.valueOf(ratioTapChanger.getTargetV());
case RATIO_TARGET_V1 -> String.valueOf(ratioTapChanger.getTargetV());
```
or do I skip line for each enum?
```java
case RATIO_TARGET_V, 
         RATIO_TARGET_V1, 
         RATIO_TARGET_V2 -> String.valueOf(ratioTapChanger.getTargetV());
```